modules/root/root_api.rb

```diff
@@ -14,7 +14,7 @@ class Proxy::RootApi < Sinatra::Base
     content_type :json
     enabled_plugins = ::Proxy::Plugins.instance.select { |p| p[:state] == :running && p[:class].ancestors.include?(::Proxy::Plugin) }
     modules = Hash[enabled_plugins.map { |plugin| [plugin[:name].to_s, plugin[:version].to_s] }].reject { |key| key == 'foreman_proxy' }
-    {:version => Proxy::VERSION, :modules => modules}.to_json
+    {:version => Proxy::VERSION, :modules => modules, :uuid => ::Proxy::SETTINGS[:instance_uuid]}.to_json
```

Comment:
The version endpoint is a public unauthenticated API. Don't we consider the instance UUID as something private?

Reply:
I wouldn't say the instance uuid is something private. The signature is.

Reply:
Personally I don't see this as a private information. It's random, so it only allows correlation, not identification. The version is IMHO much more sensitive as it allows to find out known vulnerabilities quicker.